cwltool/factory.py

```diff
@@ -1,16 +1,23 @@
 from __future__ import absolute_import
 
+# move to a regular typing import when Python 3.3-3.6 is no longer supported
+import functools
 import os
+import sys
 from typing import Callable as tCallable  # pylint: disable=unused-import
 from typing import Any, Dict, Tuple, Union
 
 from typing_extensions import Text  # pylint: disable=unused-import
 # move to a regular typing import when Python 3.3-3.6 is no longer supported
 
 from . import load_tool
-from .context import LoadingContext, RuntimeContext
+from .argparser import arg_parser
+from .context import LoadingContext, RuntimeContext, getdefault
 from .executors import SingleJobExecutor
-from .process import Process
```

Comment:
`from .process import Process` is needed for some type annotations in this file, please restore it